JSONDemoApp/Network/NetworkManager.h

```diff
@@ -8,10 +8,12 @@
 
 #import <Foundation/Foundation.h>
 
-NS_ASSUME_NONNULL_BEGIN
 
 @interface NetworkManager : NSObject
 
++ (NetworkManager *)sharedInstance;
+- (void) requestApiData: (NSString *)endPoint completionHandler:(void (^)(NSData *data, NSURLResponse *response, NSError *error))completionHandler;
```

Comment:
add empty line between class and instance methods

Reply:
Done